### PR TITLE
Update snap.js

### PR DESCRIPTION
**Pull request by Garcello822**
Adds a weird alias and makes description slightly better grammar

### CODE REVIEW

File: commands/game/snap.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ const { getEmoji } = require("../../config")
 
 module.exports = {
     name: "snap",
-    description: "Kill all players you disguised before.",
-    aliases: ["clap"],
+    description: "Kill all players you have disguised.",
+    aliases: ["clap"],["comeinlikeawreckingball"],
```
Comment (thewilloftheshadow) [inline commands/game/snap.js:7]: ["clap", "comeinbla"]

**Review by thewilloftheshadow — CHANGES_REQUESTED** [review]

File: commands/game/snap.js (hunk 1 above)
Comment (Garcello822) [reply commands/game/snap.js:7]: Bla meaning finish the sentence and remove the comma or shorten the alias?

Comment (thewilloftheshadow) [reply commands/game/snap.js:7]: i just didn't want to type the whole thing. the formatting needs to be `[` and `]` on the very ends, and separated by commas inside

Comment (Garcello822) [reply commands/game/snap.js]: Gimme a sec

Comment (Garcello822) [reply commands/game/snap.js]: I'll fix that

**Review by thewilloftheshadow — APPROVED** [review]

**Pull request merged**